### PR TITLE
Allows HTML retrieval without inserting straight into the document

### PR DESCRIPTION
**Pull request by seanmtracey**
The user can now request the HTML which is copied into the clipboard, rather than the bodies of any open messages.

### CODE REVIEW

File: src/dialog.html [hunk 1]
```diff
@@ -44,6 +44,25 @@
...
+			left: 100%;
+		}
+
+		*[data-clickable="false"]{
+			cursor: not-allowed;
+		}
+
+		#copyToClipboard{
+			position: fixed;
+		    right: 5px;
```
Comment (JakeChampion) [inline src/dialog.html]: Mixed spaces with tabs here

File: src/dialog.html [hunk 2]
```diff
@@ -121,6 +140,8 @@
 			</div>
 
 		</form>
+		<a href="#" id="copyToClipboard" data-clickable="true">Copy HTML to Clipboard</a>
```
Comment (JakeChampion) [inline src/dialog.html]: I think this would be more semantic as a button element

File: src/scripts/main.js [hunk 3]
```diff
@@ -23,36 +55,21 @@ function populateSignatures(data, force) {
 		}
 		signature.setAttribute('href', 'http://ftsig');
 		
-
 		Promise.resolve(data || getPopupInfo())
 		.then(data => {
+			debugger;
```
Comment (JakeChampion) [inline src/scripts/main.js]: remove this?

**Comment by seanmtracey** [comment]
Resolves #7 and should resolve #9

**Comment by railsagainstignorance** [comment]
LGTM. Can we get this live?

**Comment by seanmtracey** [comment]
Not quite yet. I want to get the changes the how we insert the signature in threads finished first, Then merge this into it.

**Comment by JakeChampion** [comment]
LGTM

**Pull request merged**